### PR TITLE
Allow walking in diagonal directions.

### PR DESCRIPTION
**Pull request by DaxChen**

### CODE REVIEW

**Comment by kevinshen56714** [comment]
Can you try vx^2+vy^2 = speed^2?

File: src/characters/Player.ts [hunk 1]
```diff
@@ -16,27 +16,30 @@ export default class Player extends Phaser.Physics.Arcade.Sprite {
...
-      this.setVelocity(-speed, 0)
-    } else if (cursors.right?.isDown) {
+    let vx = 0
+    let vy = 0
+    if (cursors.left?.isDown) vx -= speed
+    if (cursors.right?.isDown) vx += speed
+    if (cursors.up?.isDown) vy -= speed
+    if (cursors.down?.isDown) vy += speed
+    this.setVelocity(vx, vy)
+
```
Comment (kevinshen56714) [inline src/characters/Player.ts:29]: let vx^2+vy^2 == speed^2?

**Review by kevinshen56714 — CHANGES_REQUESTED** [review]
Let vx^2+vy^2 == speed^2?

File: src/characters/Player.ts (hunk 1 above)
Comment (DaxChen) [reply src/characters/Player.ts:30]: Done.

**Review by kevinshen56714 — APPROVED** [review]
Looks great!

**Pull request merged**